src/tree/RevisionItem.ts

```diff
@@ -33,11 +36,14 @@ export class RevisionItem implements RevisionsItemModel {
     }
 
     getTreeItem(): TreeItem {
-        const description = !this.revision.active ?
-            localize('inactive', 'Inactive') :
-            this.revision.name === this.containerApp.latestRevisionName ?
-                localize('latest', 'Latest') :
-                undefined;
+        const description = this.revisionsMode === KnownActiveRevisionsMode.Multiple ?
+            !this.revision.active ?
+                localize('inactive', 'Inactive') :
+                this.revision.name === this.containerApp.latestRevisionName ?
+                    localize('latest', 'Latest') :
+                    undefined :
```

Comment:
The number of ternaries here:

![image](https://github.com/microsoft/vscode-azurecontainerapps/assets/40250218/85e69576-b39c-45a4-878b-7d7840b8c0db)

😂